test/integration/api/integrations_controller_test.rb

```diff
@@ -158,7 +158,6 @@ def test_update_proxy_rule_position
 
   test 'deploy is never called when saving proxy info for proxy pro users' do
     rolling_updates_on
-    Account.any_instance.stubs(:provider_can_use?).with(:api_as_product).returns(false)
```

Comment:
This change belongs to https://github.com/3scale/porta/pull/2288

Reply:
But this PR is supposed to remove all `provider_can_usse` usages 🤷‍♂️  even if this test will be removed, there's this smaller step first